examples/text_labels_and_annotations/autowrap_demo.py

```diff
@@ -1,5 +1,11 @@
 """
-Auto-wrapping text demo.
+==================
+Auto-wrapping text
+==================
+
+Matplotlib can wrap text automatically, but if it's too long, the text will be
+displayed slightly outside of the boundaries of the axis anyways.
+
```

Comment:
A quick grep suggests that the common style in the codebase is an empty line after the closing quotes, not before.

Also applies below.